src/events/_listener.js

```diff
@@ -37,36 +33,52 @@ module.exports = function eventBusListener (inventory, req, res) {
 
     // @queues
     if (url === '/queues') {
-      message.arcType = 'queue'
-      let queue = get.queues(message.name)
-      message.lambda = queue
+      message.arcType = 'queues'
+      message.lambda = get.queues(message.name)
     }
     // @events
     else if (url === '/events' || url === '/') {
-      message.arcType = 'event'
-      let event = get.events(message.name)
-      message.lambda = event
+      message.arcType = 'events'
+      message.lambda = get.events(message.name)
     }
     // Who knows
-    else end
+    else end(message.name)
```

Comment:
hah I always wondered about why the function was never invoked here

Reply:
😬